src/DynamoDBMessageRepository/DynamoDBMessageRepositoryTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace EventSauce\DynamoDBMessageRepository;
+
+use Aws\DynamoDb\DynamoDbClient;
+use EventSauce\DoctrineMessageRepository\DummyAggregateRootId;
+use EventSauce\EventSourcing\AggregateRootId;
+use EventSauce\EventSourcing\Serialization\ConstructingMessageSerializer;
+use EventSauce\MessageRepositoryTestTooling\MessageRepositoryTestCase;
+use Ramsey\Uuid\Uuid;
+
+class DynamoDBMessageRepositoryTest extends MessageRepositoryTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $client = $this->client();
+
+        $tables = $client->listTables();
+        if (in_array($this->tableName, $tables->get('TableNames'), true)){
```

Comment:
NOTE: Dynamodb doesn't have an API to check if the table exists or to clear all the data in a table